python/nav/web/devicehistory/utils/history.py

```diff
@@ -208,7 +208,7 @@ def group_history_and_messages(history, messages, group_by=None):
 def describe_search_params(selection):
     data = {}
     for arg, model in (
-        ('room__location', Location),
+        ('loc', Location),
```

Comment:
Why is this changed?

Reply:
Explanation is here: https://github.com/Uninett/nav/pull/3361#issuecomment-2854584977